machAeroTutorials/aero_gridRefinementStudy.rst

```diff
@@ -66,8 +90,130 @@ An example of grid convergence plot for a family of RAE 2822 Airfoil meshes is i
 
     Figure 1: Grid convergence plot for RAE 2822 Transonic Airfoil.
 
+Pros:
+    - The grid is coarsened uniformly, giving the most mathematically rigorous convergence study, which is important for justifying solutions in your scholarly articles.
+
+Cons:
+    - To generate enough points to make a line (at least three), the finest mesh (L0) has to be extremely fine for 3D meshes to have a coarse mesh that is still in the asymptotic regime since for the ``n``th level, it needs to have :math:`(2^3)^n` fewer cells assuming a refinement ratio of 2.
```

Comment:
I think you need a space after ``n`` because now the rendering is not working. I would also move `th` to the expoent